plotly/tests/test_core/test_tools/test_trace_factory.py

```diff
@@ -100,3 +100,180 @@ def test_more_kwargs(self):
             'type': 'scatter',
             'mode': 'lines', }
         self.assertEqual(quiver, expected_quiver)
+
+
+class TestOHLC(TestCase):
+
+    def test_unequal_o_h_l_c_length(self):
```

Comment:
:cow2: As is well put, [a foolish consistency is the hobgoblin of little minds](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds). It'd be fine to call this:

`test_unequal_ohlc_length`

:smile_cat: